packages/ui-components/src/components/toggle-button/toggle-button.tsx

```diff
@@ -65,24 +67,29 @@ export class KvRadioButton implements IRadioButton, IRadioButtonEvents {
 			<Host>
 				<a
 					class={{
-						'radio-button': true,
-						'radio-button--checked': this.checked,
-						'radio-button--disabled': this.disabled,
-						'radio-button--only-icon': hasIcon && !hasLabel
+						'toggle-button': true,
+						'toggle-button--checked': this.checked,
+						'toggle-button--disabled': this.disabled,
+						'toggle-button--only-icon': hasIcon && !hasLabel
 					}}
-					part="radio-button"
+					part="toggle-button"
 					onClick={this.onClick}
 					download={this.download}
 					href={this.href}
 					target={this.target}
 				>
+					{this.withRadio && (
+						<div class="toggle-button-icon" part="toggle-icon">
```

Comment:
We have a `KvRadio`, can we use it here? If the styles do not match, please fix the component.
Also, revisit this class and part names 😄

Reply:
I'll look into it :pray: